week-1/Homework/mandatory/4-tax.js

```diff
@@ -17,25 +23,39 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function formatCurrency() {}
+function formatCurrency(number) {
+  var myNum = (number + number * 0.2).toFixed(2);
+  var sign = "£";
+  var result = sign + myNum;
+  return result;
+}
```

Comment:
If you had paid attention to the instruction, you would have known what you have been asked.
no need to rewrite the calculation of the tax as you have a function did that for you.